lib/yi_appium_caps_util.rb

```diff
@@ -24,12 +24,22 @@ def update (caps_file_name: './appium.txt', platformName_value: nil)
       end
     end
 
-    def create (platform_name)
+    def create (platform_name, form_factor)
+      
+      if (form_factor.downcase != "handheld" && 
```

Comment:
Do we have a null check for form_factor.downcase? If not, can we use Ruby's safe navigation operator? (&.!=)

Reply:
We will not get a case where formfactor is null because if it is not set we will pass a default value here 
`YiAppiumCapsUtil.create(options[:create_caps].downcase, "handheld")` and if an empty string is passed it it will reach this error
```
puts "ERROR: formFactor: '#{form_factor}' is not supported (handheld, tablet, 10foot)"
exit
```